sdk/digitaltwins/Azure.DigitalTwins.Core/tests/DigitalTwinRelationshipTests.cs

```diff
@@ -272,11 +272,11 @@ public async Task Relationships_PaginationWorks()
 
                 // create floor twin
                 string floorTwin = TestAssetsHelper.GetFloorTwinPayload(floorModelId);
-                await client.CreateDigitalTwinAsync(floorTwinId, floorTwin).ConfigureAwait(false);
+                await client.CreateDigitalTwinAsync<object>(floorTwinId, floorTwin).ConfigureAwait(false);
 
                 // Create room twin
                 string roomTwin = TestAssetsHelper.GetRoomTwinPayload(roomModelId);
-                await client.CreateDigitalTwinAsync(roomTwinId, roomTwin).ConfigureAwait(false);
+                await client.CreateDigitalTwinAsync<object>(roomTwinId, roomTwin).ConfigureAwait(false);
```

Comment:
This seems a bit odd. I expected that your PR changes would make it so that these tests/samples would pass in BasicDigitalTwins to CreateDigitalTwinAsync, not strings. Are you sure this is working as expected?

Reply:
I was expecting something like: 

```csharp
var roomTwin = new BasicDigitalTwin(roomTwinId); //Initialize the Id and other mandatory fields, too
BasicDigitalTwin returnedTwin = await client.CreateDigitalTwinAsync<BasicDigitalTwin>(roomTwinId, roomTwin).ConfigureAwait(false);
```

Reply:
That's actually what I started doing in the morning. I added <object> for now to build successfully. But I am in the process of getting rid of all <object>s

Reply:
I synched with Tim offline regarding this. I will be changing the create APIs to take in typed objects instead of a string, and serialize in the CL.